src/languages/en.ts

```diff
@@ -2196,6 +2196,18 @@ export default {
                             return 'Importing your QuickBooks Online data';
                         case 'startingImport':
                             return 'Importing your QuickBooks Online data';
+                        case 'quickbooksOnlineSyncTitle':
```

Comment:
you're planning to use the constant defined in `CONST.ts` here?

Reply:
I wasn't planing to refactor, for some reason we haven't been using CONST 🤷